src/battle/battle.h

```diff
@@ -132,7 +132,7 @@ typedef struct ActorBlueprint {
     /* 0x04 */ char unk_04;
     /* 0x05 */ u8 type;
     /* 0x06 */ u8 level;
-    /* 0x07 */ u8 maxHP;
+    /* 0x07 */ s16 maxHP;
     /* 0x08 */ s16 partCount;
     /* 0x0A */ char unk_0A[2];
```

Comment:
```suggestion
    /* 0x09 */ s16 partCount;
    /* 0x0B */ char unk_0B;
```

Reply:
hmm, wouldn't the compiler create some extra padding for either `maxHP` or `partCount` since they're misaligned after the s8->s16 change, making this not the exact same size? Not sure how to easily check that 🤔

Reply:
on the whole it doesn't really matter whether we increase struct sizes or not , but I'd rather avoid it where possible and use a more efficient field order

Reply:
> could we replace unk_04 with maxHP to avoid this?
yea, seems like that adds up, changed that now